tools/bugzilla-reports-tool/arrivals_vs_resolved.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+from datetime import datetime
+
+from helpers import *
```

Comment:
It is a good practice to avoid blind imports. Please refactor the code to avoid it.